azure-webapp-maven-plugin/README.md

```diff
@@ -244,10 +244,12 @@ Some valid regions are listed as below. All valid regions please refer to [Azure
 
 #### `<pricingTier>`
 Some valid pricing tiers are listed as below. All valid pricing tiers please refer to [Azure App Service Plan Pricing](https://azure.microsoft.com/en-us/pricing/details/app-service/).
-- `F1`
+- `F1` (Windows Only)
 - `P1V2`
 - `P2V2`
 
+> Note: Pricing tier selected should support the 'AlwaysOn' setting for deployment to succeed.
```

Comment:
I think we could prompt users that some pricing tier (like F1) didn't support Always on and may meet exceptions while deploying, as users don't know which pricing tier support `Always on`